src/core/trulens/core/session.py

```diff
@@ -176,6 +179,170 @@ def __init__(self, connector: Optional[DBConnector] = None, **kwargs):
             **self_args,
         )
 
+    def App(self, *args, app: Optional[Any] = None, **kwargs) -> App:
```

Comment:
return type is not registering since this method overrides `App`

Reply:
fixed